resources/views/projects.blade.php

```diff
@@ -6,11 +6,107 @@
 @section('content')
 <div class="padded content">
 	<h1>Projects</h1>
+	<!-- Filters -->
+	<form class="ui form">
+		<!-- Search by Name -->
+		<div class="field">
+			<label for="searchName">Name</label>
+			<input id="searchName" type="text" name="searchName" placeholder="e.g. A cool project" onkeyup="filterProjects()">
+		</div>
+		<!-- Search by Tag -->
+		<div class="field">
+			<label for="tags">Tags</label>
+			<select id="searchTags" name="tags" class="ui fluid search dropdown searchTags" multiple onchange="filterProjects()">
+				@foreach ($tags as $tag)
+					<option value={{ $tag->id }}>{{ $tag->name }}</option>
+				@endforeach
+			</select>
+		</div>
+		<button id="searchButton" type="button" class="ui primary submit button" onclick="filterProjects()">Search</button>
+	</form>
+	<!-- Project Cards -->
 	<div class="ui four stackable cards">
-		@for ($i = 0; $i < 10; $i++)
-			@projectCard(['projectImage' => 'https://source.unsplash.com/400x300/?project', 'projectName' => 'A cool project', 'category' => 'Videogames', 'projectShortDescription' => 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.', 'skillset' => ['Skill 0', 'Skill 1', 'Skill 2'], 'publicMilestone' => 'shipping'])
+		@foreach ($projects as $project)
+			@projectCard(['id'=> $project['id'],'projectImage' => $project['photo'], 'projectName' => $project['name'], 'projectShortDescription' => $project['short_description'], 'skillset' => $project->skills, 'labels' => $project->labels, 'publicMilestone' => 'shipping'])
 			@endprojectCard
-		@endfor
+		@endforeach
 	</div>
 </div>
 @endsection
+@section('scripts')
+<script>
+	$('.ui.dropdown').dropdown();
+
+	let projects = {!! $projects !!};
+
+	function filterProjects(){
```

Comment:
Throttle this function

Reply:
Esta es una manera sumamente ineficiente de hacer el filtrado. Mejor lo checamos más tarde.

Reply:
Let's remove this from this PR